index.bs

```diff
@@ -1301,7 +1301,7 @@ This is equivalent to the following infix notation:
 
 Each output sample is equal to the sum of a sample of the first input image item shifted to the left by 8 and of a sample of the second input image item. This can be viewed as a bit depth extension of the first input image item by the second input image item. The first input image item contains the 8 most significant bits and the second input image item contains the 8 least significant bits of the output reconstructed image item which has a bit depth of 16, something that is impossible to achieve with a single [=AV1 image item=].
 
-NOTE: If the first input image item is the [=primary image item=] and is enclosed in an <code>'[=altr=]'</code> group (see [[#altr-group]]) with the [=Sample Transform Derived Image Item=], the first input image item is also a backward-compatible 8-bit regular coded image item that can be used by readers that do not support [=Sample Transform Derived Image Items=] or do not need extra precision.
+NOTE: If the first input image item is the [=primary image item=] and is enclosed in an <code>'[=altr=]'</code> group (see [[#altr-group]]) with the [=Sample Transform Derived Image Item=], the first input image item is also a backward-compatible 8-bit regular coded image item that can be used by readers that do not support [=Sample Transform Derived Image Items=] or do not need extra precision. The second input image item should be a hidden image item as recommended in [[!HEIF]].
```

Comment:
Does HEIF have a general recommendation on derived image inputs being marked hidden?

Since this is not going through ISO, I don't think we have as strong requirements on what can/can't go into a NOTE, but ISO forbids `shall`/`should` in a note. Not sure if we should reformulate this somehow or move it out of the note?

Reply:
> Does HEIF have a general recommendation on derived image inputs being marked hidden?

No. There is just the definition:

```
3.1.15
hidden image
image (3.1.17) that is not intended to be displayed
```

To me it is an implicit recommendation that images not intended to be displayed should be marked as hidden.  
I am also happy to change `should be a hidden image item as recommended in [HEIF]` into `should be a hidden image item as specified in [HEIF]` or `should be a [=hidden image item=]`. Any preference?

> Since this is not going through ISO, I don't think we have as strong requirements on what can/can't go into a NOTE, but ISO forbids `shall`/`should` in a note. Not sure if we should reformulate this somehow or move it out of the note?

Moved it out but to another note and reformulated.